### PR TITLE
Options changed with select2 4

### PR DESCRIPTION
**Pull request by selckin**

### CODE REVIEW

**Comment by solomax** [comment]
Thanks!

File: select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java [hunk 1]
```diff
@@ -58,7 +57,7 @@
 
 	private AjaxSettings ajax;
 	private String data;
-	private String tags;
+	private Boolean tags;
```
Comment (martin-g) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Any reason why it should be a **B**oolean and not **b**oolean?

Comment (solomax) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Other boolean properties are Boolean
not sure why is it necessary

Comment (martin-g) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Unless `null` is a required third value I don't see a reason to use **B**oolean.

Comment (solomax) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: I'll try to check documentation for default values and fix this (no estimates unfortunately :(( )

Comment (solomax) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Actually all Boolean options are JS Objects
so I believe it is safe to change all Boolean to be boolean.

I'll change it right now

**Pull request merged**